paths/electron/starting-with-electron/03-git-repository.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Creating a Git Repository
+permalink: /electron/starting-with-electron/init-git-repository
+next-page: /electron/starting-with-electron/create-app-files
+facilitator: false
+sidebar:
+  nav: "electron-getting-starting"
+main-content: |
+
+  We have a great start of an Electron app. But, it's not being tracked with version control. Now, it's time to track this repository with Git, include a .gitignore file that fits this project, and push this up to a remote on GitHub.
```

Comment:
I should've been more specific @brianamarie. My suggestion was to remove the part where we ask people to initialize their own repo since `electron-forge` does it for them.